README.md

```diff
@@ -92,14 +101,24 @@ i.e. `name`, `surname`, `year`, `modifier` fields. You need to use two `JOIN`s
 for this query.
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT name, surname, year, modifier
+FROM students
+JOIN groups
+on students.id == groups.student_id 
```

Comment:
There is no `==` operator in SQL. Did you run this query?

Reply:
Yes, I did. And it worked.

Reply:
Actually, you are right. I don't think the operator `==` exists in all SQL dialects, but it exists in sqlite that we are using: https://www.sqlite.org/lang_expr.html#:~:text=Equals%20can%20be%20either%20%3D%20or%20%3D%3D.

Sorry for the confusion.